### PR TITLE
Fix misspellings

### PR DESCRIPTION
**Pull request by rffontenelle**
## Description

Fix misspellings reported by `codespell`.

Most fixed words are comments, with exception of `archinstall/lib/disk.py`'s `format` method which had `log_formating` keyword argument – fixed, including all calls to it.

## Bugs and Issues

```
$ codespell -ef . --skip *.pyc,.git,*.png,*.psd
./profiles/kde-wayland.py:1: environement ==> environment
./profiles/desktop.py:16: Temporarly ==> Temporarily, temporary
./profiles/desktop.py:40: therefor ==> therefore
./profiles/kde.py:1: environement ==> environment
./docs/installing/guided.rst:69: wich ==> which
./examples/guided.py:92: parition ==> partition
./examples/guided.py:123: criterias ==> criteria
./examples/guided.py:173: preperations ==> preparations
./archinstall/lib/user_interaction.py:154: indexs ==> indexes, indices
./archinstall/lib/storage.py:18: grabing ==> grabbing
./archinstall/lib/disk.py:28: missleading ==> misleading
./archinstall/lib/disk.py:290: formating ==> formatting
./archinstall/lib/profiles.py:115: shouldnt ==> shouldn't
./archinstall/lib/profiles.py:119: abolute ==> absolute
./archinstall/lib/profiles.py:159: initation ==> initiation
./archinstall/lib/profiles.py:197: shouldnt ==> shouldn't
./archinstall/lib/profiles.py:201: abolute ==> absolute
./archinstall/lib/profiles.py:213: initation ==> initiation
./archinstall/lib/output.py:9: Altough ==> Although
./archinstall/lib/installer.py:76: underlaying ==> underlying
./archinstall/lib/installer.py:271: nessecary ==> necessary
./archinstall/lib/installer.py:404: vriable ==> variable
./archinstall/lib/networking.py:59: TOOD ==> TODO
```

## How Has This Been Tested?

N/D

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code to the best of my abilities
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation where possible/if applicable

### CODE REVIEW

File: archinstall/lib/disk.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def __init__(self, path, info=None):
 		self.path = path
 		self.info = info
 		self.part_cache = OrderedDict()
-		# TODO: Currently disk encryption is a BIT missleading.
+		# TODO: Currently disk encryption is a BIT misleading.
 		#       It's actually partition-encryption, but for future-proofing this
 		#       I'm placing the encryption password on a BlockDevice level.
 		self.encryption_passwoed = None
```
Comment (pyfisch) [inline archinstall/lib/disk.py:31]: `encryption_password`?

File: archinstall/lib/networking.py [hunk 2]
```diff
@@ -56,7 +56,7 @@ def wirelessScan(interface):
 
 	storage['_WIFI'][interface]['scanning'] = True
 
-# TOOD: Full WiFi experience might get evolved in the future, pausing for now 2021-01-25
```
Comment (Torxed) [inline archinstall/lib/networking.py:59]: I feel like a drunk person reading my own code where I couldn't even spell [TODO](https://github.com/archlinux/archinstall/blame/ef021698b698037388771600e4815b8e9152872d/archinstall/lib/networking.py#L59) properly.

File: archinstall/lib/disk.py (hunk 1 above)
Comment (Torxed) [reply archinstall/lib/disk.py:31]: This makes me a bit happy, that even the best of tools don't catch some spelling mistakes ^^ Nicely spotted @pyfisch

**Comment by Torxed** [comment]
This is great, I'll merge it into master and run a few tests just to make sure those parameter changes doesn't cause any hiccups, probably won't but best to be sure :)

**Pull request merged**